manifests/params.pp

```diff
@@ -190,10 +208,14 @@
     $puppetca_path = '/usr/sbin'
     $puppetca_bin  = 'puppetca'
     $puppetrun_cmd = '/usr/sbin/puppetrun'
-  } else {
+  } elsif versioncmp($::puppetversion, '4.0') < 0 {
     $puppetca_path = '/usr/bin'
     $puppetca_bin = 'puppet cert'
     $puppetrun_cmd = '/usr/bin/puppet kick'
+  } else {
+    $puppetca_path = '/opt/puppetlabs/bin'
+    $puppetca_bin = 'puppet cert'
+    $puppetrun_cmd = '/opt/puppetlabs/bin/puppet kick'
```

Comment:
```puppet kick``` got removed in 4.x and is already deprecated in 3.x, maybe it makes sense to remove it here completely and handle it only in the foreman_proxy module.